gemrb/core/PluginLoader.cpp

```diff
@@ -98,7 +98,11 @@ static inline voidvoid my_dlsym(void *handle, const char *symbol)
 static void PrintDLError(const char *msg=NULL)
 {
 	char buffer[_MAX_PATH*2];
+#ifdef _MSC_VER
 	_strerror_s(buffer, NULL);
+#else
+	sprintf(buffer, "code %lu", GetLastError());
+#endif
```

Comment:
Looks like some MS-exclusive function. Slightly different version available in C11.

Reply:
This whole thing can be removed. While it did improve debugging, it turned out the final reports were almost as useless. Now it's potentially only adding a dependency on msvcr. But yeah, no way for you to have known.

Reply:
:+1: